text/0002-continous-integration.md

```diff
@@ -0,0 +1,71 @@
+- Start Date: 2020-03-07
+- RFC PR: https://github.com/ReCreateJS/rfcs/pull/2
+- Relevant Project: All CreateJs Projects
+
+# Continuous Integration
+
+## Summary
+
+Continuously test the CreateJS source on PRs and releases.
+
+## Motivation
+
+The practice of continuous integration gives high confidence that contributed changes do not break the project. The project can be automated to improve in quality using appropriate tooling.
+
+## Detailed design
+
+Set up TravisCI script that runs all automated forms of testing.
+
+This will include:
+
+1. Cross-browser Javascript test suite
+2. Test coverage checking
+3. Static code linting tooling with ESLint
```

Comment:
This doesn't state which ESLint rules to apply, or how they would be applied.
It might be better to split this out to it's own RFC.

Reply:
Lint rules might be tricky so I think it's a good idea to make a separate RFC.  I ran the combined build against the rules I use and a lot of things came up, such as extensive use of '==' instead of '==='.

I think the temptation would be to just adjust the rules so that the current build passes.